docs/modules/administration/pages/helm-migration.adoc

```diff
@@ -0,0 +1,31 @@
+= Migrating from Tailor to Helm
+
+Tailor has been developed for OpenShift 3.11. Back in the days, Helm 2 required the use of a privileged Tiller service and did not work well with OpenShift-specific resources. With Helm 3 and OpenShift 4, this situation as changed.
```

Comment:
s/this situation as changed/this situation has changed

Reply:
Typo fixed